socorro/processor/rules/breakpad.py

```diff
@@ -408,8 +420,32 @@ def expand_commandline(self, dump_file_path, raw_crash_path):
         }
         return self.command_line.format(**params)
 
+    def execute_process(self, command_line):
```

Comment:
This is the original `execute_process`. I figure `BreakpadStackwalkerRule2015` is deprecated, so I want to stop touching it and creating churn that could break it.